backend/users/admin.py

```diff
@@ -1,10 +1,33 @@
-from django.contrib import admin
+import csv
+from io import TextIOWrapper
+
+from django.contrib import admin, messages
 from django.contrib.auth.admin import UserAdmin
+from django.urls import path
+from django.shortcuts import render, redirect
 
-from .forms import CustomUserCreationForm, CustomUserChangeForm
+from .forms import CustomUserCreationForm, CustomUserChangeForm, BulkImportForm
 from .models import CustomUser, Faculty, Student, Department
 
 
+def add_student(record):
+    arg_values = [record['email'], record['id number'], record['name']]
+    arg_keys = ['email', 'id_num', 'name']
+    arg_dict = dict(zip(arg_keys, arg_values))
+    Student.objects.create(**arg_dict)
+
+
+def add_faculty(record):
+    arg_values = [
+        record['email'], record['name'],
+        record['psrn'], record['altername email'],
+        record['contact number'], record['department']
+        ]
+    arg_keys = ['email', 'name', 'psrn', 'alt_email', 'contact_num', 'dept']
+    arg_dict = dict(zip(arg_keys, arg_values))
+    Student.objects.create(**arg_dict)
```

Comment:
```suggestion
    Faculty.objects.create(**arg_dict)
```

Reply:
There is still no point in having two separate functions for this, considering only the fields get changed. Consider using a higher order function, or a dict to store the record mappings.